tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/ExportGroupPolicy.kt

```diff
@@ -16,9 +16,9 @@ import com.fasterxml.jackson.databind.ObjectMapper
 class ExportGroupPolicy : Runnable, RestCommand() {
     @Option(
         names = ["--save"],
-        description = ["Save the exported group policy to a specific location"]
+        description = ["Location to save the group policy file to (defaults to ~/.corda/gp/groupPolicy.json)"]
     )
-    var saveLocation: String? = null
+    var saveLocation: String = File(System.getProperty("user.home"), ".corda/gp/groupPolicy.json").path
```

Comment:
If we do this, we probably won't need the checks below (see other comment)
```suggestion
    var saveLocation: File = File(File(File(File(System.getProperty("user.home")), ".corda"), "gp"), "groupPolicy.json")
```